src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * Manages the writing and creation of files duke.txt
+ */
+public class Storage {
+    String filepath = "src/main/data/duke.txt";
```

Comment:
If this is a constant, might want to make it final and change the name to FILE_PATH.